docs/kb/semgrep-ci/bitbuket-jenkins-pipeline-projects.md

```diff
@@ -0,0 +1,95 @@
+---
+tags:
+  - Bitbucket
+  - Jenkins
+  - Pipeline projects
+description: Setting up pipeline Jenkins projects for Bitbucket repositories
+---
+
+# Setting up pipeline Jenkins projects for Bitbucket repositories
+
+## Requirements
+Follow [this guide](https://semgrep.dev/docs/kb/semgrep-ci/bitbucket-triggering-events-to-jenkins/) to setup triggering events from Bitbucket to Jenkins.
+
+## Creating a Jenkins pipeline project
```

Comment:
```suggestion
## Create a Jenkins pipeline project
```
@sebasrevuelta We **very** recently started using "simple present verbs" over "-ing" verbs, just an FYI. We're still in the process of updating the older documentation.